.gitmodules

```diff
@@ -174,4 +174,13 @@
 [submodule "vendor/golang.org/x/text"]
 	path = vendor/golang.org/x/text
 	url = https://go.googlesource.com/text
-	branch = 1e65e9bf72c307081cea196f47ef37aed17eb316
\ No newline at end of file
+	branch = 1e65e9bf72c307081cea196f47ef37aed17eb316
+[submodule "vendor/github.com/stretchr/objx"]
+	path = vendor/github.com/stretchr/objx
+	url = https://github.com/stretchr/objx.git
```

Comment:
These should all specify the branch rev like above otherwise we're not really locking our dependencies in, submodule --init will pull in the latest.